docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,31 @@
 # Adding a feed
 
-**TODO**
+ 1. To add a feed, copy the URL to RSS or Atom file's of the content wanted (for exemple, the framablog RSS URL is `https://framablog.org/feed/`).
```

Comment:
```suggestion
 1. To add a feed, copy the URL to RSS or Atom file's of the content wanted (for instance, the Framablog RSS URL is `https://framablog.org/feed/`).
```

Reply:
For example is also fine, just for ex*e*mple isn't. ^_^

Reply:
Oh yes, shame on me...
Thank you !